models/articles.js

```diff
@@ -51,3 +51,19 @@ exports.updateArticleById = async (article_id, newArticle) => {
     return Promise.reject({ status: 400, message: "Bad request" });
   }
 };
+
+exports.selectCommentsByArticleId = async (article_id) => {
+  const result = await db.query(
```

Comment:
I think you can get away without making a separate call to the db to check for the existence of the article. Think about what you'd get back from SELECT * FROM comments WHERE article_id = $1 if no comments with that article_id were found. You can definitely make things easier on yourself here!